### ISSUE TITLE
The demo is not working

### ISSUE THREAD

**Issue by dhiogoacioli** [troolee/gridstack.js#161]
I'm new here. I tried to start using the demos available, but are not working (the .js links was not in the correct reference).

I also tried to use the example that exists in the demo page, but was not resizing properly. I noticed that was the feature of 'animation' that was bugged. I do not know exactly what was missing.

I also have doubts on how to use an iframe within the divs. The size always exceeds the edges of the div, and I can not put a head above the iframe.
```
<div class="grid-stack-item" data-gs-x="0" data-gs-y="0" data-gs-width="2" data-gs-height="2"> 
  <div style="height: 15%; border: none;">HEAD 1</div>
  <iframe src="http://localhost:50272/WebForm3.aspx?test=1" style="width: 100%; height: 80%; background-color: aqua; border: none;"></iframe>
</div>
```

**Comment by FredFaust** [comment]
Hey @dhiogoacioli 
The option `animate: true` works for me. I just pass it when I initialize gridstack like that :
`var options = {
        animate: true
    };
    $('.grid-stack').gridstack(options);`

About the iframes, I think you are missing a div, if you take a closer look at the basic usage code (https://github.com/troolee/gridstack.js#basic-usage) : 

`<div class="grid-stack-item" 
        data-gs-x="4" data-gs-y="0" 
        data-gs-width="4" data-gs-height="4">
            <div class="grid-stack-item-content"></div>
    </div>`

The div with class `grid-stack-item-content` is where I insert the html that I want in my widgets. I also use that class to set a background-color with css so the background of my widgets aren't white!

Hope this helped

**Comment by dhiogoacioli** [comment]
OK, I managed to solve the issue of iframe.

Regarding the animation: I had already enabled but is catching the browser. Already tested on various browsers.

And how do I suggest that there is a head, by default (or choice) within the div 'content'?

**Comment by FredFaust** [comment]
I think that you should insert your `head` element directly into the `<div class="grid-stack-item-content"></div>` . That is the content of your widget, so you can put anything in there. I use this gridstack with Backbone.js and Marionette.js to create a view that I simply render inside the div `content`.

Can you describe what is wrong with the animation with more details, I am not sure I understand your problem.

**Comment by dhiogoacioli** [comment]
Thanks, but I found something in these libraries could help me.
On the issue of animation, it is this code I'm using:

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Float grid demo</title>

    <link href="Content/bootstrap.min.css" rel="stylesheet" />
    <link href="Scripts/gridstack.min.css" rel="stylesheet" />
    <link href="Scripts/gridstack-demo.css" rel="stylesheet" />
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">

    <script src="Scripts/lodash.min.js"></script>
    <script src="Scripts/jquery-2.1.4.min.js"></script>
    <script src="Scripts/jquery-ui-1.11.4.min.js"></script>
    <script src="Scripts/jquery.ui.touch-punch.min.js"></script>
    <script src="Scripts/gridstack.min.js"></script>
</head>
<body>
    <section id="demo" class="darklue">
        <div class="grid-stack">
            <div class="grid-stack-item" data-gs-x="0" data-gs-y="0" data-gs-width="2" data-gs-height="2">
                <div id="divConteudo1" class="grid-stack-item-content" style="background-color: aqua;">
                </div>
            </div>
            <div class="grid-stack-item" data-gs-x="2" data-gs-y="0" data-gs-width="2" data-gs-height="2">
                <div id="divConteudo2" class="grid-stack-item-content" style="background-color: aqua;">
                </div>
            </div>
        </div>
    </section>
    <script type="text/javascript">
        $(function () {
            var options = {
                always_show_resize_handle: /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent),
                float: true,
                cell_height: 80,
                vertical_margin: 10,
                animate: true
            };
            $('.grid-stack').gridstack(options);
        });

        $('#divConteudo1').load("http://localhost:50272/WebForm3.aspx?test=1");
        $('#divConteudo2').load("http://localhost:50272/WebForm3.aspx?test=2");
    </script>
</body>
</html>
```

**Comment by vert3x** [comment]
Hi @dhiogoacioli, 

Lately I used iframes with gridstack and I have an issue when I reduce the size of the window  somwhere around 740px or less, each time I go to that size the iframe gets reloaded! and I'm pretty sure it's related to gridstack because I tried using iframes in a simple HTML page and I haven't that issue. I even tried to use a simple image in the src attr of the iframe!

Have you encountred this issue ?
